lib/cmd/fh3/projects.js

```diff
@@ -112,6 +112,15 @@ function createProject(args, deployEnvironment, cb) {
     });
   }
 
+  // Like NGUI, filter out appTemplates which are 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingWhitespace)